src/encoded/commands/index_annotations.py

```diff
@@ -58,10 +58,19 @@ def run(app):
 
     # bulk index of annotations
     annotations = json_from_path(registry.settings.get('annotations_path'), {})
-    try:
-        es.bulk(index=index, body=annotations, refresh=True, request_timeout=30)
-    except:
-        print("Unable index the annotations")
+
+    base_timeout = 45
+    additional_timeout = 0
+    annotation_indexing_success = False
+
+    while not annotation_indexing_success:
+        try:
+            es.bulk(index=index, body=annotations, refresh=True, request_timeout=base_timeout+additional_timeout)
+            annotation_indexing_success = True
+        except:
+            log.error("Unable to index the annotations.", exc_info=True)
+            additional_timeout += 10
```

Comment:
Could eliminate the boolean, would be a little easier to read IMO:
```suggestion
    while True:
        try:
            es.bulk(index=index, body=annotations, refresh=True, request_timeout=base_timeout+additional_timeout)
            break
        except:
            log.error("Unable to index the annotations.", exc_info=True)
            additional_timeout += 10
```